fixes overwrite check (#7)
ginkgo-project/git-cmake-format: CMakeLists.txt

  message(WARNING "Not in a git repository")
else()
  set(GCF_GIT_DIR_SCRIPT ${GCF_GIT_HOOKS_DIR}/git-cmake-format.py)
  if((NOT GCF_FORCE_OVERWRITE) AND (EXISTS ${GCF_GIT_HOOKS_DIR}/pre-commit))

>     message(STATUS "custom hook or git-cmake-format.py already installed, skipping it (overwrite with GCF_FORCE_OVERWRITE=ON)")

Reviewer: ```suggestion
    message(STATUS "custom hook is already installed, skipping it (overwrite with GCF_FORCE_OVERWRITE=ON)")
```
